physionet-django/console/templates/console/known_referencess.html

```diff
@@ -0,0 +1,27 @@
+{% extends "console/base_console.html" %}
+
+{% load static %}
+
+{% block title %}PhysioNet Users{% endblock %}
+
+{% block local_css %}
+<link rel="stylesheet" type="text/css" href="{% static 'custom/css/pagination.css' %}">
+{% endblock %}
+
+{% block content %}
+<script src="{% static 'custom/js/cookie.js' %}"></script>
+<script src="{% static 'project/js/dynamic-formset.js' %}"></script>
+<script src="{% static 'console/js/search-console.js' %}"></script>
```

Comment:
Should these be in `{% block local_js_top %}` or does it not really matter? Only thing I can find, otherwise looks good! 👍

Reply:
I would say the search and cookie yes because of the search done, but the formset is prob not needed. I will remove it